src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/prisonperson/profiledetails/ProfileDetailsNomisApiService.kt

```diff
@@ -7,7 +7,7 @@ import org.springframework.web.reactive.function.client.awaitBody
 import uk.gov.justice.digital.hmpps.prisonertonomisupdate.nomissync.model.UpsertProfileDetailsRequest
 import uk.gov.justice.digital.hmpps.prisonertonomisupdate.nomissync.model.UpsertProfileDetailsResponse
 
-@Service
+@Service("prisonPersonProfileDetailsNomisApiService")
```

Comment:
Had to qualify the prison person beans that I've copied. Looking forward to the prison person cull.